pkg/client/k8s/v1alpha1/unstructured.go

```diff
@@ -64,6 +61,33 @@ func (k kind) resource() (resource string) {
 	return
 }
 
+// isNamespaced flags if the kind is namespaced or not
+//
+// NOTE:
+//  This may not be the best of approaches to flag a resource as namespaced or 
+// not. However, this fits the current requirement. This might need a revisit 
+// depending on future requirements.
+func (k kind) isNamespaced() (no bool) {
```

Comment:
Another way could be to have a map which is created in the init() or somewhere.

Reply:
I will have a look in future requirements @princerachit